recommendsmay2016github-chicago.md

```diff
@@ -0,0 +1,5 @@
+## Things to do in Riverwoods
+
+### Great Places in Eat
+
+### Fun THings to do
```

Comment:
The 'H' in Things is capitalized. Otherwise this file looks awesome :+1: :100: @may2016github